Formula/knot-resolver.rb

```diff
@@ -1,10 +1,18 @@
 class KnotResolver < Formula
   desc "Minimalistic, caching, DNSSEC-validating DNS resolver"
   homepage "https://www.knot-resolver.cz"
-  url "https://secure.nic.cz/files/knot-resolver/knot-resolver-1.4.0.tar.xz"
-  sha256 "ac19c121fd687c7e4f5f907b46932d26f8f9d9e01626c4dadb3847e25ea31ceb"
+  revision 2
```

Comment:
JFTR the `revision 2` is reference to bump to `revision 1` in #18735

Reply:
So that means this is blocked on #18735.

Reply:
Yeah, but #18735 is a simple version bump, so I don't expect it to take long.

Reply:
That has been merged now, so it could go in. I'll rebase on current master and re-push.